pages/pridejse.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Přidej se
+permalink: /pridejse
+---
+
+# Přidej se k nám!
+
+Studuješ na Fakultě informačních technologií a chceš poznat aktivní FIŤáky nebo jenom najít odreagování od Progtestu? 
+
+Tak jsi tady správně!
+
+Jsme klub FIT++, který působí na Fakultě informačních technologií ČVUT. Pořádáme akce jako [Hello FIT](https://klubfitpp.cz/hellofit), FIT párty, deskovky na fakultě, FITripy, LAN párty a spoustu další edukačních nebo zábavných eventů, na kterých se můžeš nejenom dozvědět knowlage nad rámec předmětů ve škole, ale hlavně poznat spoustu aktivních lidí z fakulty.
+
+**Proč bych se měl\* přidat?**
+
+- Rozšíříš si kontakty, poznáš nové kamarády nejenom z FIT ale i z dalších fakult nebo studentských spolků
+- Dostaneš přednostní možnost zapojit se do pomoci fakultě na eventech jako COFIT, DOD,.., které jsou zaplacené
+- Na edukačních eventech si můžeš rozšířit obzory mimo hranice předmětů
+- Můžeš si vyzkoušet organizaci eventu - softskill, který se fakt hodí
+- Aktivita ve studentském spolku je ceněná v CVčku
+- Jako dobrovolník dostaneš **zdarma klubové tričko**
+- Odneseš si dobrý pocit, že jsi udělal něco dobrého pro FIŤáckou komunitu
+
+**Jak se můžu zapojit?**
+
+- Přijď na jakoukoliv akci, kterou najdeš [tady v kalendáři](/events)
+- Sleduj channel [help-needed](https://discord.com/channels/808276641603649607/1288795322524635176) na Discordu a zapoj se do projektu, který hledá dobrovolníky nebo pracovníky
+- Napiš na Discord, že máš nápad, a my ti ho pomůžeme zrealizovat
+- Přijď na zasedání Představenstva a společně něco vymyslíme
+- Kandiduj do Představenstva a buď nositelem myšlenky
+
+**Co tam v tom klubu vlastně děláte?**
```

Comment:
V ostatních "kapitolách" máme body s velkými písmeny na začátku a tady jsou najednou malá, asi by se hodilo mít to všude stejně.